src/plugins/saved_objects/README.md

```diff
@@ -0,0 +1,119 @@
+# Saved object
+
+The saved object plugin provides all the core services and functionalities of saved objects. It is utilized by many core plugins such as [`visualizaton`](../visualizations/), [`dashboard`](../dashboard/) and [`wizard`](../wizard/), as well as external plugins.
```

Comment:
A short description of what saved objects are would be very useful here. e.g. 

> They are the primary way to store or persist app/plugin data in a standardized form in OpenSearch Dashboards. They allow a plugin developer to manage creating, saving, editing and retrieving data that the application needs. They can also reference other saved objects and have useful features out of the box like migrations and strict typing. They can also be managed using the Saved Object Management UI.

This is just what came off the top of my head. I'd have someone like @vagimeli or @joshuarrrr correct the language though.

Reply:
Added the description. Thanks!!